types/stylelint/index.d.ts

```diff
@@ -324,6 +324,16 @@ declare module 'stylelint' {
 			 * @internal
 			 */
 			createLinter: (options: LinterOptions) => InternalApi;
+			/**
+			 * Resolves the effective configuation for a given file. Resolves to
+			 * `null` if no config is found.
+			 * @param filePath The path to the file to get the config for.
+			 * @param options The options to use when creating the Stylelint instance.
```

Comment:
[suggestion] Same as above.

```suggestion
			 * @param filePath - The path to the file to get the config for.
			 * @param [options] - The options to use when creating the Stylelint instance.
```

Reply:
The brackets aren't used in TSDoc since the type definition already specifies whether or not the parameter is optional, but I'll add the hyphens.